src/components/picker/Picker.driver.new.ts

```diff
@@ -3,8 +3,14 @@ import {TextFieldDriver} from '../textField/TextField.driver.new';
 import {ModalDriver} from '../modal/Modal.driver.new';
 import {DialogDriver} from '../../incubator/Dialog/Dialog.driver.new';
 import {ButtonDriver} from '../button/Button.driver.new';
+import {ExpandableOverlayDriver} from '../../incubator/expandableOverlay/ExpandableOverlay.driver';
+
+export const PickerDriver = (props: ComponentProps, useDialog: boolean) => {
```

Comment:
I don't like this addition of special input prop `useDialog` - I think we had that added somewhere before and you objected as well...